gui/src/callbacks.cc

```diff
@@ -496,6 +537,18 @@ void create_scroom(PresentationInterface::Ptr presentation)
   g_signal_connect (static_cast<gpointer>(drawingArea), "button-release-event", G_CALLBACK (on_button_release_event), view.get());
   g_signal_connect (static_cast<gpointer>(drawingArea), "scroll-event", G_CALLBACK (on_scroll_event), view.get());
   g_signal_connect (static_cast<gpointer>(drawingArea), "motion-notify-event", G_CALLBACK (on_motion_notify_event), view.get());
+
+
+  char * uriList = strdup("text/uri-list");
```

Comment:
What is the right way to instantiate such a GtkTargetEntry from a constant string in C++...?

Reply:
And ping @kees-jan

Reply:
Looks like gtk makes a copy of anything it needs. Valgrind reports this one as a leak :wink:

Reply:
It wouldn't compile without that on my machine. It's a quirky interaction between C and C++ here. But indeed I forgot to free the data again...

Reply:
I, too, had some complaints about it not being allowed to convert from string literals to char pointers. But the compiler does seem to accept the char array I'm currently using.